### PR TITLE
Feature/add month signature

### PR DESCRIPTION
**Pull request by aagm**
great thanks for this marvel :D

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 node_modules
+.env
+yarn.lock
```
Comment (tiagojsag) [inline .gitignore]: :(

Comment (aagm) [reply .gitignore]: :heart:

**Comment by davidsingal** [comment]
it works! please merge :)

**Comment by tiagojsag** [comment]
remove yarn.lock from .gitignore please

**Pull request merged**